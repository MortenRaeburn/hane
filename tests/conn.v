Definition impl : forall P : Prop, forall Q : Prop, Prop :=
    fun P : Prop => fun Q : Prop => forall p : P, Q.

Definition and : forall P : Prop, forall Q : Prop, Prop :=
    fun P : Prop => fun Q : Prop => forall R : Prop, impl (impl P (impl Q R)) R.