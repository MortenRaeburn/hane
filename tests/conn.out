Definition impl : forall[Prop] (forall[Prop] (Prop)) := fun[Prop] (fun[Prop] (forall['1] ('1))).
Definition and : forall[Prop] (forall[Prop] (Prop)) := fun[Prop] (fun[Prop] (forall[Prop] (((impl) (((impl) ('2)) (((impl) ('1)) ('0)))) ('0)))).
